core/src/main/java/com/microsoft/applicationinsights/internal/util/DeviceInfo.java

```diff
@@ -55,4 +55,11 @@ public static String getHostName() {
     public static String getLocale() {
         return Locale.getDefault().toLanguageTag();
     }
+
+    public static String getJavaCacertsPath() {
```

Comment:
I don't think this quite fits here, ok to create `SSLUtils` in this package, and refactor out (parameterize) the common code that is throwing the exception could go in `SSLUtils` also.